client/components/questionnaire1.js

```diff
@@ -1,19 +1,36 @@
 import React, {Component} from 'react'
+import SubQuestions from './subcategories-questions'
 
 class Questionnaire1 extends Component {
   constructor(props) {
     super(props)
     this.handleSubmit = this.handleSubmit.bind(this)
     this.handleChange = this.handleChange.bind(this)
     this.state = {
-      income: '',
-      housing: '',
-      utilities: ''
+      food: {
+        name: 'Food',
+        overallMonthly: ''
+      },
+      utilities: {
+        name: 'Utilities',
+        overallMonthly: ''
+      },
+      shopping: {
+        name: 'Shopping',
+        overallMonthly: ''
+      }
     }
   }
 
   handleChange = event => {
-    this.setState({...this.state, [event.target.name]: event.target.value})
+    this.setState({
+      //this works for now but maybe refactor to use this.setState({prevState}) instead of ...this.state
+      ...this.state,
+      [event.target.name.toLowerCase()]: {
```

Comment:
The category name may get very tricky with multi-word categories, maybe add the category database attribute name as a different key on the questionnaire array?

Reply:
I did write sort of a helper function to handle that issue in the questionnaire2.js component where there were some multi-word categories. I could apply it to all the questionnaire components for consistency, but since that was the only one that had multi-words, there wasn't really a need to.